rex/renewable_resource.py

```diff
@@ -65,8 +65,8 @@ def _get_SAM_df(self, ds_name, site):
 
         return res_df
 
-    def _preload_SAM(self, sites, tech='pvwattsv7', clearsky=False,
-                     bifacial=False, means=False):
+    def _preload_SAM(self, sites, tech='pvwattsv7', time_step_size=None,
```

Comment:
How come SAMResource doesn't have any methods like this? You can't actually retrieve a SAM resource DF with just the SAMResource handler, can you?

Reply:
SAMResource is currently only a container for the resource data that is used by SAM to pull the needed data. We pre-load all of the data needed for SAM and the class handles extracting the sites of interest for each worker.

Reply:
Do you think it would make sense to add a method to SAMResource to load real data? Or at least change the docstrings to describe how to use this? I would be super confused if I just wanted to use SAMResource to run SAM.